README.md

```diff
@@ -46,3 +46,8 @@ Shorten the Semantic distance
 - Functional to express relation between input and output
 - Object oriented to encapsulate state with actions
 - Apect oriented to capture repeating aspects
+
+## Extensions
+
+-Extension 1: Early Warning
```

Comment:
Appreciate the documentation in README :)
It makes it easier for me...what to expect